rosetta-cli-conf/testnet/config.json

```diff
@@ -1,6 +1,6 @@
 {
   "network": {
-    "blockchain": "Avalanche",
+    "blockchain": "Camino",
     "network": "Fuji"
```

Comment:
Should we also change the `network' here?

Reply:
fixed